### ISSUE TITLE
US recovered cases not updated since 17th march. 

### ISSUE THREAD

**Issue by parthpandya19** [CSSEGISandData/COVID-19#1219]
US recovered cases not updated since 17th march
Displays 0

**Comment by ZurMaD** [comment]
It's your opportunity to upload the data. Colaboratory work please.

**Comment by r-lomba** [comment]
@ZurMaD upload how? They don't accept pull requests on CSV files (and this is correct, it would mess up everything). The point is that this is a service to the world, and if you as an institution (JHU) are not maintaining proper stats for key Countries (US and, for instance, France both have wrong recovery rates) than it's not a reliable and useful service, despite the huge and respectable work done everyday

**Comment by Amogh100** [comment]
Is there any kind of mechanisms we could introduce so that accurate data is updated in a timely manner? I understand JHU is supposed to be the source of truth on this data, and they should be given their position but it's really important this data is updated frequently with accuracy. This is important because there are many clients who are using this data as part of APIs for different websites, and that can lead to misinformation.

**Comment by ryanwoconnor** [comment]
Same as #1173

**Comment by paolinic03** [comment]
The time series dataset doesn’t include recoveries for the US but the daily files do. I just do not really understand this. What is the difference between the time series pull and the daily pull? Should be pulling from the same source just used for different use cases. All the time series set needs to do is pull in the additional US record for recoveries shown in the daily files.

**Comment by paolinic03** [comment]
Major news #1250

**Comment by TracyVargo** [comment]
I'm not understanding why recovery information is not being updated. It is definitely no less important than the other possible outcome though maybe less definitive. Active case tracking is critical (Confirmed - recoveries - deaths). Will the daily reports still have the information? If so, the lack of recovery info in the time series format will make the reports much less useful.

**Comment by r-lomba** [comment]
@TracyVargo I agree. I was producing quite informative charts tracing active cases. I would dare to say that without information about recoveries you get to lose the real point of what's going on in the world during time. I'm already looking for alternative datasets

For example, a couple of visualisations that I put up and that are considered quite unique, (I know it from the feedback I had)... And I wouldn't be able to update everyday anymore after they'll put these major changes in place:

![active_cases](https://user-images.githubusercontent.com/33959199/77351041-9052ce80-6d3d-11ea-8734-e65ba7991d18.png)

![active_cases_01](https://user-images.githubusercontent.com/33959199/77351001-87fa9380-6d3d-11ea-82d9-980c8acc85e5.png)

**Comment by paolinic03** [comment]
@TracyVargo it’s not that they decided to not update recovered and active cases because they are being cruel. It is that they have seen recovered and active case data isn’t accurate and therefore isn’t available to report on. You can’t report or update date that isn’t available.

So don’t think of it as they just are taking it out of the dataset, but that it is not available so it really can’t be continued.

**Comment by r-lomba** [comment]
@paolinic03 it has probably been a hard decision on their side, in the end they are showing integrity and care by choosing not to publish something they can't trust. Still it's a pity they will discontinue that stream

**Comment by TracyVargo** [comment]
@paolinic03 but then how are other sources reporting this data on a continuing basis and how to we correct the problem with lack of data? This isn't something that you say 'oh well' over.

**Comment by paolinic03** [comment]
@TracyVargo what other sources are reporting this data and are making it publicly available in the fashion JHU CSSE is? Most other sources are actually using JHUs datasets in there own. If you know of other original sources with this data, I would love to know because I would switch directly to them.

**Comment by TracyVargo** [comment]
@paolinic03 well, Microsoft Bing and Worldometer both have this available, but am not sure where they are pulling their info from. I just know it is continuously changing. In any case, somehow this information has to be out there and JHU CSSE should have the best of what is available. I don't know anyone can truly analyze the progress of this situation without this information.

**Comment by paolinic03** [comment]
@TracyVargo completely agree, yet here we are. I follow Worldometers but know their data is licensed.

**Comment by hopeseekr** [comment]
I've seen this problem.

I have created a 100% open source, 100% open access RESTful API for COVID_19 cases.

But I absolutely suck @ getting volunteers. It's a major problem in my life.

https://github.com/CovidResearch/CovidTracker

If you could tell me where to get the recovered data? it is ESSENTIAL to know the true death rate of this thing.

**Comment by Mevemajusun** [comment]
Where is this data being pulled from?
